transactionoutbox-spring/pom.xml

```diff
@@ -43,11 +48,6 @@
       <artifactId>spring-context</artifactId>
```

Comment:
Still need this?

Reply:
Checked, yes, it contains the `@Service` annotation.